.circleci/config.yml

```diff
@@ -7,6 +7,23 @@ defaults: &defaults
 
 version: 2
 jobs:
+  test-site:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: yarn && pip3 install -r requirements.txt
+      - run:
+          name: Build the site resources
+          command: yarn run build
+      - run:
+          name: Run the site server
+          command: ./entrypoint 0.0.0.0:80
```

Comment:
```suggestion
          command: PORT=80 yarn run serve 
```

Reply:
The reason why I didn't use `yarn run serve` in other projects is because that endpoint is meant for local development rather than *necessarily* being how it's run in production. In this case it's pretty similar (apart from not being on port 80), but it might not always be. The way the app is run in production is instead defined [in the Dockerfile](https://github.com/jkfran/jaas.ai/blob/circleci-test-site/Dockerfile#L20).